tests/ui/drop/lint-if-let-rescope-gated.rs

```diff
@@ -0,0 +1,30 @@
+// This test checks that the lint `if_let_rescope` does not action if either
+// - Edition 2024 is not enabled, or
+// - the feature gate `if_let_rescope` is disabled
```

Comment:
Problem [REV8]: we should update this test comment, namely if this is a migration late then we can't gate it on ed 2024.

Reply:
Comment updated